content/introduction.html

```diff
@@ -69,6 +69,111 @@ <h1> Preface </h1>
 future classes, as we did, in order to take future profiles to new heights.
 </p>
 
+<div class="vbx6"></div>
+
+<h1> Timeline </h1>
+<p class="full">
+  Within our time at Waterloo, the Software Class of 2021 (a.k.a SEXXI) experienced many firsts and witnessed many note
+  worhy events that we have noted for you below.
+</p>
+<!-- Timeline from https://www.sliderrevolution.com/resources/css-timeline/ -->
+<div id="timeline">
+  <div>
+  <section class="year">
+    <h3>Spring 2016</h3>
+    <section>
+      <ul>
+        <li>Harambe dies</li>
+      </ul>
+    </section>
+  </section>
+  <section class="year">
+    <h3>Fall 2016</h3>
+    <section>
+      <ul>
+        <li>SEXXI start at the University of Waterloo</li>
+        <li>First shortened Orientation Week</li>
+      </ul>
+    </section>
+  </section>
+  <section class="year">
+    <h3>Winter 2017</h3>
+    <section>
+      <ul>
+        <li>r/uwaterloo wins the meme war against r/UofT thanks to a <a href="https://www.reddit.com/r/uwaterloo/comments/5ibr95/fuck_it_if_this_post_receives_2000_upvotes_i_will/">goose tatoo</a></li>
+        <li>SEXXI use (ie: beta test) Waterlooworks</li>
+      </ul>
+    </section>
+  </section>
+  <section class="year">
+    <h3>Fall 2017</h3>
+    <section>
+      <ul>
+        <li>New Residence Building (NRB) opens on campus</li>
+        <li>NRB is renamed Claudette Milla Hall (CMH) 2 months later</li>
+      </ul>
+    </section>
+  </section>
+  <section class="year">
+    <h3>Summer 2017</h3>
+    <section>
+      <ul>
+        <li>SLC PAC Extension start</li>
+      </ul>
+    </section>
+  </section>
+  <section class="year">
+    <h3>Fall 2018</h3>
+    <section>
+      <ul>
+        <li>Lime Scooter Pilot</li>
+      </ul>
+    </section>
+  </section>
+  <section class="year">
+    <h3>Winter 2019</h3>
+    <section>
+      <ul>
+        <li>First UWaterloo Snow Day</li>
+      </ul>
+    </section>
+  </section>
+  <section class="year">
+    <h3>Summer 2019</h3>
+    <section>
+      <ul>
+        <li>ION opens</li>
```

Comment:
```suggestion
        <li>ION, Waterloo's light rail transit system, opens</li>
```

idk how necessary it is to explain this